tools/Environments/DevHome.Environments/Views/LandingPage.xaml

```diff
@@ -259,7 +269,7 @@
                     <!--  'Sync' button -->
                     <Button Padding="40 5 30 8" Command="{x:Bind ViewModel.SyncButtonCommand}" Margin="0 3 0 0">
                         <StackPanel Orientation="Horizontal">
-                            <FontIcon FontFamily="{StaticResource SymbolThemeFontFamily}" Glyph="&#xE72C;"/>
+                        <FontIcon FontFamily="{StaticResource SymbolThemeFontFamily}" FontSize="14" Glyph="&#xE72C;"/>

```

Comment:
This page will have merge conflicts. Christian said this is 16.

Reply:
No worries, I made this change before the update. Will remove

Reply:
I just made all of the resources use the "+" instead. I figure its better for us to fix all at once with a common template